docs/spec/historical.rst

```diff
@@ -280,10 +280,10 @@ and type checkers may warn if they are used.
 --------------------------
 
 Before Python 3.10 (:pep:`604`), Python did not support the ``|`` operator
-for creating unions of types. Therefore, the ``typing.Union`` special form can also
+for creating unions of types. Therefore, the ``typing.Union`` :term:`special form` can also
 be used to create union types. Type checkers should treat the two forms as equivalent.
 
-In addition, the ``Optional`` special form provides a shortcut for a union with ``None``.
+In addition, the ``Optional`` :term:`special form` provides a shortcut for a union with ``None``.
```

Comment:
I'm not sure I'd call this a "shortcut" since `Optional[Foo]` is longer than `Foo | None`. :)

Reply:
It is shorter than `Union[Foo, None]` though, so it should be sufficient to clarify `typing.Union` instead of "union". It is part of the historical section and `typing.Union` was mentioned in the paragraph just before. I'd also clarify `typing.Optional`, or drop the prefix in both paragraphs, it's weird to use a prefix in the paragraph above but not below.

Reply:
Yes, I'm pretty sure this was originally meant as a "shortcut" relative to `Union[T, None]`, though even that is only a little longer than `Optional[T]`. I'll revise the sentence to merely note the equivalence.